bridges/primitives/chain-bridge-hub-cumulus/src/lib.rs

```diff
@@ -54,6 +54,14 @@ const MAXIMUM_BLOCK_WEIGHT: Weight = Weight::from_parts(constants::WEIGHT_REF_TI
 	.saturating_div(2)
 	.set_proof_size(polkadot_primitives::MAX_POV_SIZE as u64);
 
+/// We allow for 2 seconds of compute with a 6 second average block.
+///
+/// This is a copy-paste from the `polkadot-sdk` repo where all system parachains are switched to async backing (https://github.com/paritytech/polkadot-sdk/pull/2949).
```

Comment:
> This is a copy-paste from the `polkadot-sdk` repo

This is confusing. Isn't this PR also on `polkadot-sdk`?

Reply:
Changed